### DISCUSSION TITLE
Using with Netlify

### DISCUSSION THREAD

**Discussion by tjperry07** [UmbrellaDocs/linkspector#51] [General]
What is your recommendation for the baseUrl when using something like Netlify and Vercel where the preview links are randomly generated?

I was thinking to use this action I could:
- Either just tell everyone to run it locally before making a PR?
- Have it run as a cron job against the site?

**Comment by gaurav-nelson** [comment]
Hello @tjperry07 

Do you mean, in your site generator you have links like `/docs/one.md` which in the final site points to `<some-temp-url>.vercel.app/docs/one.md`. However, in you code base you don't have `./docs/one.md` in the root of your project. 

Can you provide an example (or repository) so that I can understand it better?

**Reply by tjperry07** [reply]
https://github.com/tjperry07/public-product-documentation/pull/45 Just a random PR. But you can see that the deploy preview changes each time. In Netlify case its incremental. 

<img width="960" alt="Screenshot 2024-05-21 at 9 48 00 AM" src="https://github.com/UmbrellaDocs/linkspector/assets/5667652/1261388e-7cf4-45ef-8d53-11ccca971d3d">
